neonvm/controllers/virtualmachine_controller.go

```diff
@@ -637,6 +637,7 @@ func podSpec(virtualmachine *vmv1.VirtualMachine) (*corev1.Pod, error) {
 			Annotations: virtualmachine.Annotations,
 		},
 		Spec: corev1.PodSpec{
+			EnableServiceLinks:            virtualmachine.Spec.ServiceLinks,
```

Comment:
would it trigger errors when `virtualmachine.Spec.ServiceLinks` node defined (`null`)

Reply:
It shouldn't. If you don't pass this field, it is already null because this is the default value of the bool pointer. So, nothing should change